service.go

```diff
@@ -221,6 +221,19 @@ func NewGeckoDriverService(path string, port int, opts ...ServiceOption) (*Servi
 	return s, nil
 }
 
+// NewIEDriverService starts a IEDriverServer instance in the background.
+func NewIEDriverService(path string, port int, opts ...ServiceOption) (*Service, error) {
```

Comment:
To properly support this, this needs tests, which is not a small undertaking:

1. A new top-level test function needs to be added to selenium_test.go, called `TestInternetExplorer`, similar to the other tests in that file.
2. Somehow, we'll need to skip that test if it isn't executed on Windows.
3. We should add a run on Travis in Windows environment.